client/components/domains/register-domain-step/index.jsx

```diff
@@ -584,20 +583,36 @@ const RegisterDomainStep = React.createClass( {
 				}
 				break;
 
-			case 'mappable_but_forbidden_subdomain':
+			case 'not_mappable_forbidden_subdomain':
```

Comment:
I couldn't find any reference where the string can be set to this. Can we clean up cases as such?